at_client/lib/src/service/sync_service_impl.dart

```diff
@@ -396,6 +395,7 @@ class SyncServiceImpl implements SyncService, AtSignChangeListener {
             ExceptionScenario.remoteVerbExecutionFailed, e.message));
         _logger.severe(
             'Exception occurred in fetching sync response : ${e.getTraceMessage()}');
+        rethrow;
```

Comment:
assume commitIds 100-150 from server has to be synced in batches of 10 commitIds each.
If there is an exception at commitId 120..121-150 will not be synced from server. is this behavior fine?

Reply:
Up until Jul 29, the exception was not being caught - i.e. if an exception was encountered, it was thrown. I am simply reverting to that behaviour. I don't believe it is correct to continue in this case when an exception has occurred (note that the exception is occurring **when getting the response to the sync: command** and not when processing results)